src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueBenchmarkToast.java

```diff
@@ -12,7 +12,6 @@ public class SeedQueueBenchmarkToast implements Toast {
 
     private boolean finished;
     private boolean fadeOut;
-    private long fadeOutStart;
```

Comment:
Am I understanding correctly that getting rid of `fadeOutStart` and the logic using it means the toast will just immediately disappear on reset? If thats true, I'm not sure I want that

Reply:
> Am I understanding correctly that getting rid of `fadeOutStart` and the logic using it means the toast will just immediately disappear on reset? If thats true, I'm not sure I want that

it will not, it will animate away as it did (:

Reply:
Oh I think I remember now, i suppose `fadeOutStart` wasn't the best field name lol